### PR TITLE
Created a text file with important etiquette while working on OSPs

### PR DESCRIPTION
**Pull request by PranavReddy0148**
#1 Summarized the provided MDN doc on OSP etiquette in a text file📄.

### CODE REVIEW

**Comment by akhilender-bongirwar** [comment]
@PranavReddy0148, can you summarize in your own words.

**Comment by PranavReddy0148** [comment]
@akhilender-bongirwar, I have updated the text file as required.

**Review by akhilender-bongirwar — APPROVED** [review]
Good that you have accepted that you used AI previously. LGTM :+1:
